src/portfolios/portfolios.service.ts

```diff
@@ -42,13 +38,7 @@ export class PortfoliosService {
         return await identity.portfolios.getPortfolio();
```

Comment:
`.catch`

I can see the case for `try/catch` to avoid repeating `.catch` call here, but I still think multiple `.catch` is neater than `try/catch` block.